SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -58,18 +57,13 @@ public class BootstrapperSettings : IBootstrapperSettings
 
         #region Constructor(s)
 
-        public BootstrapperSettings(AnalysisPhase phase, IEnumerable<string> childCmdLineArgs, string sonarQubeUrl, LoggerVerbosity verbosity, ILogger logger)
+        public BootstrapperSettings(AnalysisPhase phase, IEnumerable<string> childCmdLineArgs, LoggerVerbosity verbosity, ILogger logger)
         {
-            if (sonarQubeUrl == null)
```

Comment:
it's not needed to handle sonarqube url at all in the bootstrapper (it was before, when the bootstrapper downloaded the rest of the scanner from sonarqube). 
It will be parsed and checked in the begin step.